javascript/Readme.md

```diff
@@ -1,5 +1,159 @@
+Finciero Javascript Style Guide
+===============================
+
+We based our style guide on the [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript). In this document we only specify rules in which we disagree.
+
+## Rules
+
+- Avoid using trailing commma. eslint: [`comma-dangle`](http://eslint.org/docs/rules/comma-dangle.html).
+
+```javascript
+// bad
+const hero = {
+  firstName: 'Dana',
+  lastName: 'Scully',
+};
+
+const heroes = [
+  'Batman',
+  'Superman',
+];
+
+// good
+const hero = {
+  firstName: 'Dana',
+  lastName: 'Scully'
+};
+
+const heroes = [
+  'Batman',
+  'Superman'
+];
+
+```
+
+- Avoid declaring function that receive more than 3 arguments. If you need to pass more arguments, pass them as an object.
+
+```javascript
+// bad
+function foo(var1, var2, var3, var4) {
+// ...
+}
+
+// good
+function foo(args) {
+// ..
+}
+```
+
+- When an Arrow Function receive only one paramters should not be surrounded with parenthesis.
+
+```javascript
+// bad
+[1, 2, 3].map(function (x) {
+  const y = x + 1;
+  return x * y;
+});
+
+// better
+[1, 2, 3].map((x) => {
+  const y = x + 1;
+  return x * y;
+});
+
+// good
+[1, 2, 3].map(x => {
+  const y = x + 1;
+  return x * y;
+});
+```
+
+- We use special rultes for using the `let` variable declaration. All variables declared with let must be declared at the beginning of the scope.
```

Comment:
typo, should said: `spetial rules`:trollface:, also `let`should be mark as special variable with '`' provided by markdown